Kerberos.NET/Dns/DnsQueryWin32.cs

```diff
@@ -38,7 +38,8 @@ IntPtr pReserved
         private static readonly HashSet<int> IgnoredErrors = new HashSet<int>()
         {
             9002, // server not found
-            9003, // record not found
+            9003, // record not found,
+            9501, // server not found
```

Comment:
Is this mapping to  DNS_INFO_NO_RECORDS ?

Reply:
Yes, adding comment to indicate.